optuna/integration/skopt.py

```diff
@@ -31,24 +31,8 @@
 class SkoptSampler(BaseSampler):
     """Sampler using Scikit-Optimize as the backend.
 
-    Example:
-
-        Optimize a simple quadratic function by using :class:`~optuna.integration.SkoptSampler`.
-
-        .. testcode::
-
-            import optuna
-
-
-            def objective(trial):
-                x = trial.suggest_float("x", -10, 10)
-                y = trial.suggest_int("y", 0, 10)
-                return x**2 + y
-
-
-            sampler = optuna.integration.SkoptSampler()
-            study = optuna.create_study(sampler=sampler)
-            study.optimize(objective, n_trials=10)
+    The use of SkoptSampler is highly not recommended, as the development of Scikit-Optimize
```

Comment:
How about adding a link?

```suggestion
    The use of :class:`~optuna.integration.SkoptSampler` is highly not recommended, as the development of Scikit-Optimize
```

Reply:
Thank you for your review! I deliberately did not make it a link. Do you think we should add a link even from the same documentation page?

Reply:
I think the decorative notation makes it easier to read. Or, replacing it with "this sampler" is OK.

Reply:
Sure. Let me apply your suggestion :pray: